Cargo.toml

```diff
@@ -15,7 +15,7 @@ members = [
   "tests/rust/roundtrips/*",
   "tests/rust/test_*",
 ]
-exclude = ["examples/rust/revy"]
+exclude = ["examples/rust/revy", "examples/rust/robby_fischer"]
```

Comment:
can we call it `chess_robby_fisher` so it's easier to find?

Reply:
Fixed.